physics/README.md

```diff
@@ -0,0 +1,44 @@
+# Physics
+This discipline of science is quite different and even nascent with regards to the lab. However, we are open to contributions and collaborations.
+
+## Special Relativity
+The below mentioned resources help one understand the special theory of relativity, from scratch and do not really require any pre-requisites.
+
+### Course(s)
+#### [Advance Course on STR by Dr. H. C. Verma](https://www.youtube.com/playlist?list=PL8g67naApM8i21RucyrHYqyVwmurwA1UC)
```

Comment:
Writing a full form is better I guess

Reply:
It becomes a bit too long. But fine, make the change